rsrc/menus/BladesOfExile.rc

```diff
@@ -32,9 +32,9 @@ LANGUAGE LANG_ENGLISH, SUBLANG_ENGLISH_US
 
 // Icon with lowest ID value placed first to ensure application icon
 // remains consistent on all systems.
-IDI_BLADESOFEXILE       ICON                    "../../../rsrc/icons/win/BOE Icon.ico"
-IDI_BOESAVE             ICON                    "../../../rsrc/icons/win/BoE Save.ico"
-IDI_BOESCEN             ICON                    "../../../rsrc/icons/win/BoE Scenario.ico"
+IDI_BLADESOFEXILE       ICON                    "BOE Icon.ico"
+IDI_BOESAVE             ICON                    "BoE Save.ico"
+IDI_BOESCEN             ICON                    "BoE Scenario.ico"
```

Comment:
This _might_ break the vs2013 project, but it will need to be addressed with the one is moved anyways.